scripts/jenkins/ardana/ansible/inventory

```diff
@@ -18,7 +18,7 @@ qe102           ansible_host=10.84.81.113
 qe201           ansible_host=10.84.81.33
 qe202           ansible_host=10.84.81.49
 qe2023          ansible_host=10.84.89.151
-scale           ansible_host=10.84.48.51
+qescale         ansible_host=10.84.43.33
 
```

Comment:
Hi Flavio a question about how we decide this IP. In https://gitlab.suse.de/cloud-qe/ardana-bm-info/blob/master/qescale/networks.yml the cidr for external-api is 10.84.43.32/28 ... can you please point me to the code which reserves this IP so that it is not used by any other node for external-api?

Reply:
The way that ardana reserve IPs is sequential (starting from the deployer). So, by ensuring that the deployer will be on the external-api network (see [1]), we also make sure that it will always get the first available IP from the external-api network range.

1. https://github.com/SUSE-Cloud/automation/blob/master/scripts/jenkins/ardana/ansible/roles/input_model_generator/vars/templates/interface/compact.yml

Reply:
OK thanks Flavio, got it.